docker/docker-compose.test.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.4"
+
+services:
+  api_tests:
```

Comment:
would try to understand how (and what type of) `pytest-django` does in terms of database setup and teardown (drops the schema? creates it on demand? rollbacks a transaction?)